meetups/index.md

```diff
@@ -5,18 +5,29 @@ title: Meetups
 ---
 
 * [35th Meetup](https://www.meetup.com/PyData-London-Meetup/events/240813803/)
-  * Richard J Brooker on AI & ART - Style Transfer
-  * Saeed Amen on Using Python in financial markets
-  * Won Bae Suh on Arms for Sale: UK Bazzar in the 21st Century
+  * Richard J Brooker on [AI & ART - Style Transfer](https://www.youtube.com/watch?v=U1vb-NI9sKE)
+  * Saeed Amen on [Using Python in financial markets](https://www.meetup.com/PyData-London-Meetup/events/240813803/comments/481296769/)

```

Comment:
I think a direct link to the slides would be good, maybe followed with a short link or set of links to the extra material. Happy for you to link to either the comment or a bunch of short links to the content directly.

My only concern with the comment is comments can get deleted. On the other hand, extra links might get added to it later. \shrug